integration-cli/docker_cli_help_test.go

```diff
@@ -51,19 +52,14 @@ func (s *DockerSuite) TestHelpTextVerify(c *check.C) {
 		helpCmd := exec.Command(dockerBinary, "help")
 		helpCmd.Env = newEnvs
 		out, ec, err := runCommandWithOutput(helpCmd)
-		if err != nil || ec != 0 {
```

Comment:
hum, right, didn't see those, maybe make sense to modify dockerCmd with env []string, @vdemeester what do you think?